### PR TITLE
VCB-186 Fix: allow multiple labels on DCD instruction

### PR DESCRIPTION
**Pull request by ni507**
Test Code

```
AREA |.data|, DATA, READWRITE
                    

ADDR_LED_15_0           EQU     0x60000100
ADDR_LED_31_16          EQU     0x60000102
ADDR_7_SEG_BIN_DS1_0    EQU     0x60000114
ADDR_DIP_SWITCH_15_0    EQU     0x60000200
ADDR_HEX_SWITCH         EQU     0x60000211

NR_CASES        		EQU     0xB
BIT_MASK				EQU		0xFFF0

jump_table
						DCD   case_dark, case_add, case_minus
					
    
AREA |.text|, CODE, READONLY
; -------------------------------------------------------------------
; -- Main
; -------------------------------------------------------------------   
                        
MOVS R0, #1
MOVS R1, #1
MOVS R2, #1
LSLS  	R2, #2  
LDR   	R7, =jump_table 
LDR   	R7, [R7, R2] 
BX    	R7

; STUDENTS: To be programmed

case_dark       
                LDR  	R0, =0

case_add        
                ADDS 	R0, R0, R1
case_minus        
                ADDS 	R0, R0, R1

```

### CODE REVIEW

**Comment by deflox** [comment]
Maybe you could add a test for this in the tests for the encoder? Then we know that this case is covered and we do not break it again by accident.

File: src/assembler/encoder.ts [hunk 1]
```diff
@@ -209,8 +209,7 @@ function isDataInstruction(instruction: IInstruction): boolean {
 function isSymbolDataInstruction(instruction: IInstruction): boolean {
   return (
     instruction.name == 'DCD' &&
```
Comment (deflox) [inline src/assembler/encoder.ts:211]: Isn't this comma separated behavior also possible with `DCW` and `DCB`?

Comment (clooudch) [reply src/assembler/encoder.ts:211]: I don't think so. A symbol has always a value of 32 bits (word). So, it doesn't make much sense to only use `DCB` or `DCW`. But I haven't tried it out on the actual CT board yet.

**Review by clooudch — APPROVED** [review]

File: src/assembler/encoder.ts (hunk 1 above)
Comment (m-dot-wigger) [reply src/assembler/encoder.ts:211]: grafik
does only work with DCD

**Comment by ni507** [comment]
> Maybe you could add a test for this in the tests for the encoder? Then we know that this case is covered and we do not break it again by accident.

@deflox Added a test in the encoder tests.

**Review by deflox — APPROVED** [review]

**Pull request merged**